folding-schemes/src/frontend/utils.rs

```diff
@@ -152,14 +152,14 @@ impl<F: PrimeField> FCircuit<F> for CustomFCircuit<F> {
 /// than the one done in the `AugmentedFCircuit`, but without adding all the extra constraints
 /// of the AugmentedF circuit logic, in order to run lighter tests when we're not interested in
 /// the the AugmentedF logic but in the wrapping of the circuits.
-#[cfg(test)]
+// #[cfg(test)]
 pub struct WrapperCircuit<F: PrimeField, FC: FCircuit<F>> {
     pub FC: FC, // F circuit
     pub z_i: Option<Vec<F>>,
     pub z_i1: Option<Vec<F>>,
 }
 
-#[cfg(test)]
+// #[cfg(test)]
```

Comment:
my bad, I disabled it to run some local tests using the lib as a dependency to ensure that everything works fine, so needed that circuit exposed to external usage. But just [re-enabled](https://github.com/privacy-scaling-explorations/sonobe/compare/27ab0e2a41f62d7c658f2a0450332593878d429b..2bb66c1fcdfd19e9bfa7b516f5becfa9782104ae) it again.

Reply:
my bad, this is because in the crate frontends, the WrapperCircuit is used for some tests, so with exposing it we avoid duplicating it. Updated it to [completely remove the lines](https://github.com/privacy-scaling-explorations/sonobe/compare/2bb66c1fcdfd19e9bfa7b516f5becfa9782104ae..b42999f507c823e97fa1361be3b04986d6804a36)